app/models/piece.rb

```diff
@@ -32,6 +41,24 @@ def move_to!(x, y)
     raise ArgumentError, "Can't move piece" unless update_attributes(x_coordinate: x, y_coordinate: y)
   end
 
+  # testing potential moves - move will not be saved
+  def move_to(x, y)
```

Comment:
I ended using the move_to! and it worked when I moved the piece back.  I'd love help getting this move_to to work.